organization/src/main/scala/com/improving/app/organization/domain/Organization.scala

```diff
@@ -1,6 +1,206 @@
 package com.improving.app.organization.domain
 
-object Organization{
+import akka.actor.typed.scaladsl.Behaviors
+import akka.actor.typed.{ActorRef, Behavior}
+import akka.cluster.sharding.typed.scaladsl.EntityTypeKey
+import akka.pattern.StatusReply
+import akka.persistence.typed.PersistenceId
+import akka.persistence.typed.scaladsl.{Effect, EventSourcedBehavior, ReplyEffect}
+import com.google.protobuf.timestamp.Timestamp
+import com.improving.app.common.domain.MemberId
+import com.improving.app.common.errors.Validation._
+import com.improving.app.common.errors._
+import com.improving.app.organization.domain.OrganizationValidation._
 
+import java.time.Instant
+
+object Organization {
+  val TypeKey = EntityTypeKey[OrganizationRequestEnvelope]("Organization")
+
+  case class OrganizationRequestEnvelope(request: OrganizationRequestPB, replyTo: ActorRef[StatusReply[OrganizationEvent]])
```

Comment:
Why have the PB suffix?

Reply:
Otherwise there is a name collision. I tried OrganizationRequestMessage but then scalapb creates a type called OrganizationRequestMessageMessage

Reply:
Hmm.. is there no better name then? I'm afraid that the PB suffix will lose its meaning over time and become "vestigial" (I don't know a better word to describe it)

Reply:
I was using it by analogy to the PBMessageSerializable trair which is another type we already have in the system and many of the protobuf messages extend

Reply:
I think that's alright, it makes sense to me